tests/testthat/test-crunch-data-frame.R

```diff
@@ -216,4 +206,13 @@ with_test_authentication({
         expect_equal(ds_df$mr_3, as.vector(ds$MR$mr_3))
         expect_equal(ds_df$v4, as.vector(ds$v4))
     })
+
+    test_that("CDF with assigned variables returns dataframe", {
+        ds_df$new_local_var <- 4:1
+        ds_df$new_local_var2 <- 1:4
+        # and we can get a dataframe from this new one.
+        true_df <- as.data.frame(ds_df)
+        expect_equal(as.vector(ds_df$new_local_var), 4:1)
+        expect_equal(as.vector(ds_df$new_local_var2), 1:4)
```

Comment:
I might be missing something, but it doesn't look like you're asserting anything about `true_df` here?